packages/react-ui/src/app/builder/builder-page.tsx

```diff
@@ -30,55 +53,77 @@ const BuilderPage = () => {
     <div className="flex h-screen w-screen flex-col">
       {run && <RunDetailsBar run={run} onExitRun={ExitRun} />}
       <BuilderNavBar />
       <ResizablePanelGroup direction="horizontal">
-        {leftSidebar !== LeftSideBarType.NONE && (
-          <>
-            <ResizablePanel
-              id="left-sidebar"
-              defaultSize={25}
-              order={1}
-              minSize={25}
-              maxSize={47}
-            >
-              {leftSidebar === LeftSideBarType.RUNS && <FlowRecentRunsList />}
-              {leftSidebar === LeftSideBarType.RUN_DETAILS && (
-                <FlowRunDetails />
-              )}
-              {leftSidebar === LeftSideBarType.VERSIONS && <FlowVersionsList />}
-            </ResizablePanel>
-            <ResizableHandle withHandle={true} />
-          </>
-        )}
+        <>
+          <ResizablePanel
+            id="left-sidebar"
+            defaultSize={0}
+            minSize={0}
+            maxSize={45}
+            order={1}
+            ref={leftHandleRef}
+            className={cn('min-w-0', {
+              [minWidthOfSidebar]: leftSidebar !== LeftSideBarType.NONE,
+              [animateResizeClassName]: !isDraggingHandle,
+            })}
+          >
+            {leftSidebar === LeftSideBarType.RUNS && <FlowRecentRunsList />}
+            {leftSidebar === LeftSideBarType.RUN_DETAILS && <FlowRunDetails />}
+            {leftSidebar === LeftSideBarType.VERSIONS && <FlowVersionsList />}
+          </ResizablePanel>
+          <ResizableHandle
+            disabled={leftSidebar === LeftSideBarType.NONE}
+            withHandle={leftSidebar !== LeftSideBarType.NONE}
+            onDragging={setIsDraggingHandle}
+          />
+        </>
+
         <ResizablePanel defaultSize={100} order={2} id="flow-canvas">
           <ReactFlowProvider>
             <FlowCanvas />
           </ReactFlowProvider>
         </ResizablePanel>
-        {rightSidebar !== RightSideBarType.NONE && (
-          <>
-            <ResizableHandle withHandle={true} />
 
-            <ResizablePanel
-              id="right-sidebar"
-              defaultSize={35}
-              maxSize={60}
-              minSize={30}
-              order={3}
-            >
-              {rightSidebar === RightSideBarType.PIECE_SELECTOR && (
-                <PiecesCardList />
-              )}
-              {rightSidebar === RightSideBarType.PIECE_SETTINGS && (
-                <StepSettings key={flowVersion.id + selectedStep?.stepName} />
-              )}
-            </ResizablePanel>
-          </>
-        )}
+        <>
+          <ResizableHandle
+            disabled={rightSidebar === RightSideBarType.NONE}
+            withHandle={rightSidebar !== RightSideBarType.NONE}
+            onDragging={setIsDraggingHandle}
+          />
+
+          <ResizablePanel
+            ref={rightHandleRef}
+            id="right-sidebar"
+            defaultSize={0}
+            minSize={0}
+            maxSize={45}
```

Comment:
Please revert it to 60% this intentional changes from previous design.

Reason: I compared the design with previous one, It's much better than this way and all other similar designs / platforms allow you for bigger right sidebar, specially for code settings sidebar, which is now easier to use in the smaller screen.